client/components/Modals/ModalContainer.js

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { ModalContextCreator } from './ModalContext';
+
+/*
+  maps a string to a modal child,
+  used to determine which modal child is to be rendered
+  e.g:
+  const MODAL_COMPONENTS = {
+    'CREATE_EVENT': CreateEventModal,
+  };
+ */
+const MODAL_COMPONENTS = {
```

Comment:
Could you have this on one line? `const MODAL_COMPONENTS = {}`. It looks like it's expected to have a value or what does this do?

Reply:
Yes it's expected to values, that is why it's opened.